pkg/reconciler/taskrun/resources/taskresolution.go

```diff
@@ -14,16 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1_test
+package resources
 
 import (
-	"testing"
-
-	"github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1"
-	"knative.dev/pkg/webhook/resourcesemantics"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 )
 
-func TestTypes(t *testing.T) {
-	// Assert that types satisfy webhook interface.
-	var _ resourcesemantics.GenericCRD = (*v1alpha1.PipelineResource)(nil)
+// ResolvedTask contains all the data that is needed to execute
```

Comment:
this comment is not accurate - there is no mention of a TaskRun here and no reference to a resource

Reply:
Thanks, I also moved it to `pkg/reconciler/taskrun/resources/taskspec.go` since the original file is mostly removed.